repo2docker/buildpacks/conda/__init__.py

```diff
@@ -396,6 +396,7 @@ def get_env_scripts(self):
                     echo auth-none=1 >> /etc/rstudio/rserver.conf && \
                     echo auth-minimum-user-id=0 >> /etc/rstudio/rserver.conf && \
                     echo "rsession-which-r={env_prefix}/bin/R" >> /etc/rstudio/rserver.conf && \
+                    echo "rsession-ld-library-path={env_prefix}/lib" >> /etc/rstudio/rserver.conf && \
```

Comment:
Looking at the docs in https://docs.posit.co/ide/server-pro/1.0.25/r-sessions.html (ctrl-f 'rsession-ld-library-path'), this *adds* rather than replaces elements in `LD_LIBRARY_PATH` so this is good to go it think.